bot.py

```diff
@@ -273,9 +284,7 @@ async def on_raw_reaction_add(reaction):
             embed.set_author(
                 name=message.author.display_name, icon_url=message.author.avatar
             )
-            embed.set_footer(
-                text=f"{message.guild.name} | {message.channel.name}"
-            )
+            embed.set_footer(text=f"{message.guild.name} | {message.channel.name}")
```

Comment:
Stop changing the formatting randomly. It just makes it harder to review changes.